.github/workflows/pr-examples-build.yml

```diff
@@ -0,0 +1,42 @@
+name: "PR Examples Build"
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'examples/**'
+jobs:
+  build:
+    name: Build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+        include:
+          - os: ubuntu-latest
+            testAdditionalJavaVersions: true
```

Comment:
We don't currently have this set up for examples, but maybe we should doesn't hurt to leave it